Formula/xorg-server.rb

```diff
@@ -0,0 +1,70 @@
+class XorgServer < Formula
+  desc "X Window System display server"
+  homepage "https://www.x.org"
+  url "https://gitlab.freedesktop.org/xorg/xserver.git",
+    tag:      "xorg-server-21.1.3",
+    revision: "85397cc2efe8fa73461cd21afe700829b2eca768"
+  license "MIT"
+
+  depends_on "font-util"   => :build
+  depends_on "libxkbfile"  => :build
+  depends_on "meson"       => :build
+  depends_on "ninja"       => :build
+  depends_on "pkg-config"  => :build
+  depends_on "util-macros" => :build
+  depends_on "xkbcomp"     => :build
+  depends_on "xorgproto"   => :build
+  depends_on "xtrans"      => :build
+
+  depends_on "libapplewm"
+  depends_on "libxfixes"
+  depends_on "libxfont2"
+  depends_on "mesa"
+  depends_on "pixman"
+  depends_on "xcb-util"
+  depends_on "xcb-util-image"
+  depends_on "xcb-util-keysyms"
+  depends_on "xcb-util-renderutil"
+  depends_on "xcb-util-wm"
+
+  on_linux do
+    depends_on "libepoxy"
+    depends_on "libxshmfence"
+  end
+
+  def install
+    meson_args = std_meson_args + %W[
+      -Dxephyr=true
+      -Dxf86bigfont=true
+      -Dxcsecurity=true
+
+      -Dxkb_dir=#{HOMEBREW_PREFIX}/share/X11/xkb
+      -Dxkb_bin_dir=#{Formula["xkbcomp"].opt_bin}
+      -Dxkb_output_dir=#{HOMEBREW_PREFIX}/X11/xkb/compiled
+
+      -Dbundle-id-prefix=homebrew.mxcl
+      -Dbuilder_addr=#{tap.remote}
+      -Dbuilder_string=#{tap.name}
+    ]
+    # macOS doesn't provide `authdes_cred` so `secure-rpc=false`
+    # glamor needs GLX enabled `libepoxy` on macOS
+    if OS.mac?
+      meson_args += %W[
+        -Dsecure-rpc=false
+        -Dapple-applications-dir=#{libexec}
+      ]
+    end
+
+    system "meson", "build", *meson_args
+    system "meson", "compile", "-C", "build"
+    system "meson", "install", "-C", "build"
+    x = OS.mac? ? "Xquartz" : "Xorg"
+    bin.install_symlink bin/x => "X"
```

Comment:
Isn't the preferred way to do this via `xinit`?

Reply:
Linux implementation [creates the symlink](https://gitlab.freedesktop.org/xorg/xserver/-/blob/master/hw/xfree86/meson.build#L141-145) during installation.
I should change it to `bin.install_symlink bin/"Xquartz" => "X" if OS.mac?`.

Reply:
In that case, yes that seems the way to do it.

Reply:
`XQuartz` create symlink in [compile script](https://github.com/XQuartz/XQuartz/blob/57c5d8aa4df91483ada70f98bf2e3a54a0b07cf5/compile.sh#L955).